packages/base/tests/blockchain.test.ts

```diff
@@ -83,3 +86,64 @@ test("a real world Omni Lock witness should work as expected", (t) => {
   t.is(omniLockWitnessPlaceholder.byteLength, 105);
   t.deepEqual(packedWitness, omniLockWitnessPlaceholder);
 });
+
+test("Script codec", (t) => {
```

Comment:
these cases are same duplicated with serialization.test.ts, maybe we can remove them